src/nrepl/jack-in.ts

```diff
@@ -262,9 +257,8 @@ export async function jackIn(connectSequence: ReplConnectSequence, cb?: () => un
 
   let projectConnectSequence: ReplConnectSequence = connectSequence;
   if (!projectConnectSequence) {
-    try {
-      projectConnectSequence = await getProjectConnectSequence();
-    } catch (e) {
+    projectConnectSequence = await getProjectConnectSequence();
```

Comment:
`getProjectConnectSequence` can possible throw an exception. Do we not still want to catch that here like we were?

Reply:
Right now this will show an error alert to the user with the error message being thrown, instead of continuing with jack-in. I'll see if it is that we shuold show our own message instead and return early.

Reply:
I now remember why I choose to do it like this. For some reason the error message is never shown when I try to do vscode.window.errorMessage. I now choose to disregard that and output the error to the output window as well as (failing to) show the error message. It works from the connect call site, so there's that.